spec/fixtures/controllers/with_string_controller.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+require_relative "base_test_controller"
+
+class WithStringController < BaseTestController
+  validate_params_for [:index, :create], format: :json do |p|
+    p.param :with_scrub, String, scrub_invalid_utf8: true
```

Comment:
why not just always scrub invalid UTF8?  why would we want to keep invalid UTF8?

Reply:
Because it's breaking change to public published library. You can set default if needed:
https://github.com/peopleforce/validate_params/pull/65/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R23-R28

A string with Invalid UTF8 characters is still a string with invalid UTF encoding. It still can be used, processed in code, and even saved into DB (depending on column type and DBMS).